vendor/bat-native-confirmations/src/bat/confirmations/internal/logging_util.cc

```diff
@@ -67,16 +68,12 @@ std::string HeadersToString(
   return base::JoinString(formatted_headers, "\n");
 }
 
-std::string UrlResponseToString(
-    const std::string& url,
-    const int response_status_code,
-    const std::string& response,
-    const std::map<std::string, std::string>& headers) {
-  const std::string formatted_headers = HeadersToString(headers);
+std::string UrlResponseToString(const ledger::UrlResponse& response) {
+  const std::string formatted_headers = HeadersToString(response.headers);
 
   return base::StringPrintf("URL Response:\n  URL: %s\n  Response "
-      "Status Code: %d\n  Response: %s\n  Headers:\n%s", url.c_str(),
-          response_status_code, response.c_str(), formatted_headers.c_str());
+      "Status Code: %d\n  Response: %s\n  Headers:\n%s", response.url.c_str(),
```

Comment:
Please change `Response:` to `Body:` to match variable name

Reply:
done